### PR TITLE
Avatar and profile encouragement

### PR DESCRIPTION
**Pull request by ProfessorTom**
I've wanted to add this to the !welcome message for a while. Here's my PR.

### CODE REVIEW

File: lita_config.rb [hunk 1]
```diff
@@ -58,6 +58,8 @@
     /^!welcome/i => <<MSG,
 Welcome to :256:! Join the conversation. There are many tech-specific channels. (e.g. #code, #testing, #jobs). A lot of silliness goes down in #random.
 
+If you haven't done so already, please upload an avatar and fill out your proflie. We're a friendly group–we don't bite, promise!–but we are a community that likes to know or neighbors!
```
Comment (RandomlyKnighted) [inline lita_config.rb]: I believe that should be 

`but we are a community that likes to know our neighbors!`

You left off the u on the word our.

Comment (ProfessorTom) [inline lita_config.rb]: Whomp! Whomp!

**Comment by iamvery** [comment]
@tech256/folks

**Comment by bryanp** [comment]
Nice, I like it 😀

**Pull request merged**